### ISSUE TITLE
As a developer, I need to know when I might have introduced a change that breaks compatibility with anndata.

### ISSUE THREAD

**Issue by maniarathi** [chanzuckerberg/single-cell#47]
**Story**: As a developer, I need my test results to provide meaningful feedback on whether my code has broken anything or caused any incompatibilities. For example, breaking the compatibility tests should give me information on how I might have made cellxgene incompatible with our most important dependency, anndata, which would cause pain later on to our users.
**Need**: Developers need reliable, running tests that tell us if we've added code that will cause dependency failures for desktop cellxgene users.

**TL;DR**: Fix the compatibility tests in cellxgene.

**Comment by tihuan** [comment]
@lesliecodes @maniarathi since we don't estimate an epic, I assume we're pointing https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/cellxgene/1865? Thank  you!

**Comment by lesliecodes** [comment]
Ah, I saw the estimate me! tag so I thought we also estimate epics. What's it there for then? lol

**Comment by maniarathi** [comment]
We did this! Yay!
